cassovary-core/src/main/scala/com/twitter/cassovary/graph/Traverser.scala

```diff
@@ -324,6 +328,10 @@ trait QueueBasedTraverser[+V <: Node] extends Traverser[V] {
     }
   }
 
+  protected def chooseNodes(node: Node): Seq[Int] = {
```

Comment:
perhaps have a more descriptive name? may be chooseNeighborsAfterVisit() or something similar...